src/TestScope.js

```diff
@@ -128,6 +132,18 @@ export default class TestScope {
     component.props.onPress();
   }
 
+  // Public: 'Focus' a component (e.g. a `<TextInput />`).
+  // Your component should respond to the property `onFocus`.
+  //
+  // identifier - Identifier for the component.
+  //
+  // Returns a promise, use await when calling this function. Promise will be
+  // rejected if the component is not found.
+  async focus(identifier) {
+    const component = await this.findComponent(identifier);
+    component.props.onFocus();
+  }
```

Comment:
This function does exactly the same thing as press, but rather focuses the component 😄